openshift-build-example/README.md

```diff
@@ -0,0 +1,32 @@
+# OpenShift Build Example
+## General
+This is an example showing how to add a BuildConfig in OpenShift, using it to build and push a new image, while collecting build-info.
```

Comment:
...how to add a BuildConfig in OpenShift, using it to build and... --> ...how to add a BuildConfig in OpenShift and use it to build and...